### PR TITLE
Address a couple of low-hanging fruits

### PR DESCRIPTION
**Pull request by retailcoder**
- Stops flagging IsMissing used with a ParamArray parameter
- Swallows any exceptions thrown from VBComponent.HasDesigner

### CODE REVIEW

**Comment by Vogel612** [comment]
Just merge it already :sweat_smile: :wink:

File: Rubberduck.VBEditor.VBA/SafeComWrappers/VB/VBComponent.cs [hunk 1]
```diff
@@ -78,13 +78,20 @@ public bool HasDesigner
...
+                    using (var designer = new UserForm(Target.Designer as VB.Forms.UserForm))
+                    {
+                        return !designer.IsWrappingNullReference;
+                    }
                 }
-                using (var designer = new UserForm(Target.Designer as VB.Forms.UserForm))
+                catch
                 {
-                    return !designer.IsWrappingNullReference;
+                    return false;
```
Comment (bclothier) [inline Rubberduck.VBEditor.VBA/SafeComWrappers/VB/VBComponent.cs:94]: Same comment RE: logging/generic catchall

**Review by bclothier — APPROVED** [review]
Great work and looks good; just a minor nitpick.

File: Rubberduck.VBEditor.VB6/SafeComWrappers/VB/VBComponent.cs [hunk 2]
```diff
@@ -103,13 +103,20 @@ public bool HasDesigner
...
+                    using (var designer = new UserForm(Target.Designer as VB.VBForm))
+                    {
+                        return !designer.IsWrappingNullReference;
+                    }
                 }
-                using (var designer = new UserForm(Target.Designer as VB.VBForm))
+                catch
                 {
-                    return !designer.IsWrappingNullReference;
+                    return false;
```
[the remark this answers is not in the document]
Comment (retailcoder) [reply Rubberduck.VBEditor.VB6/SafeComWrappers/VB/VBComponent.cs:119]: That's a good point and I did forget about the existence of the `protected static readonly Logger` field in the base class: this logging could easily be added without introducing any new dependencies.

[the remark this answers is not in the document]
Comment (retailcoder) [reply Rubberduck.VBEditor.VB6/SafeComWrappers/VB/VBComponent.cs:119]: That said, if `Target.Designer` throws, we probably have bigger problems ;-)

**Pull request merged**